### DISCUSSION TITLE
Setting / knowing the screen resolution for Quest

### DISCUSSION THREAD

**Discussion by mvvvv** [StereoKit/StereoKit#844] [Q&A] [answered]
Is it possible to adjust the screen resolution on the Quest (and others)?

Looking at the logs I notice the following points:
77: The native_activity launches well for a content_rec of 3664x1920

596: StereoKit: Set view: PrimaryStereo to 2880x3168@1msaa
807: StereoKit: Set view: PrimaryStereo to 1440x1584@1msaa

The image is a little disappointing with a lot of aliasing (I saw that the subject of aliasing is already open)
[log.txt](https://github.com/StereoKit/StereoKit/files/13991826/log.txt)

**Comment by maluoi** [comment, marked as answer]
Yeah, check out `Renderer.Scaling` and `SKSettings.renderScaling`, this allows you to scale the render resolution up and down a bit! It's not a perfect replacement for MSAA, but I think MSAA is probably a "pretty soon" thing.

I don't believe I'm surfacing the current resolution anywhere though. Current resolution could be nice, but without a "native resolution" as reference, it doesn't feel as useful. Oculus's OpenXR runtime has always provided a surprisingly low resolution as their recommendation, and that's the only one I really get from OpenXR.
